.vscode/settings.json

```diff
@@ -2,5 +2,6 @@
     "[rust]": {
         "editor.defaultFormatter": "rust-lang.rust-analyzer",
         "editor.formatOnSave": true
-    }
+    },
+    "terminal.integrated.scrollback": 1000000000
```

Comment:
Hmm, this belongs in your user config, not in the project config 🙂

Reply:
It was already in the project, vscode just updated the file automatically. Should we remove the whole file?

Reply:
VSCode separates workspace (project) configs and user configs. Having Rust-specific config options in a Rust project makes sense. But the scrollback is a personal preference.

When editing settings, you can choose whether you want to change user settings or workspace settings:

![screenshot-20230903-191716](https://github.com/coredump-ch/gitcash/assets/105168/e36c8740-ea43-4da6-8641-04c569d735f7)

If you remove the added line from the config, and choose "User" below the search bar in the settings, you can change settings without them being added to this file 🙂

Reply:
I removed that commit.